gradio/data_classes.py

```diff
@@ -366,6 +366,7 @@ class Page(TypedDict):
 
 class BlocksConfigDict(TypedDict):
     version: str
+    deep_link_state: NotRequired[Literal["valid", "invalid", "none"]]
```

Comment:
What does this parameter do?

Reply:
If the URL is not valid, we return deep_link_state of invalid so that the browser can display an appropriate error message as opposed to just failing silently.